docs/operator-guides/configuring-airbyte-db.md

```diff
@@ -3,7 +3,9 @@
 Airbyte uses different objects to store internal state and metadata. This data is stored and manipulated by the various Airbyte components, but you have the ability to manage the deployment of this database in the following two ways:
 
 * Using the default Postgres database that Airbyte spins-up as part of the Docker service described in the `docker-compose.yml` file: `airbyte/db`.
-* Through a dedicated custom Postgres instance \(the `airbyte/db` is in this case unused, and can therefore be removed or de-activated from the `docker-compose.yml` file\).
+* Through a dedicated custom Postgres instance \(the `airbyte/db` is in this case unused, and can therefore be removed or de-activated from the `docker-compose.yml` file\). It's not a good practice to deploy mission-critical databases on Docker or Kubernetes. 
```

Comment:
Excluding the content in the parentheses, the first half of this paragraph is not a complete sentence:

> Through a dedicated custom Postgres instance

It's not your change, but the original sentence is incomplete. Should correct it in this PR.

Reply:
Sorry, it's a bullet point. So it does not need to be a complete sentence.